test/functional/interface_rpc.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Tests some generic aspects of the RPC interface."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal, str_to_b64str
+
+import http.client
+import json
+import urllib.parse
+
+class RPCInterfaceTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+        self.setup_clean_chain = True
+
+    def send_request(self, request):
+        self.conn.request('POST', '/', json.dumps(request), self.headers)
+        out = self.conn.getresponse().read()
+        return json.loads(out.decode('ascii'))
+
+    def test_batch_request(self):
+        self.log.info("Testing basic JSON-RPC batch request...")
+
+        request = [
+            # Two basic requests that will work fine.
+            {"method": "getblockcount", "id": 1},
+            {"method": "getbestblockhash", "id": 2},
+            # Request that will fail.  The whole batch request should still
+            # work fine.
+            {"method": "invalidmethod", "id": 3},
+        ]
```

Comment:
nit, add another call here or move one of the above?

Reply:
What other call do you mean here?  Or is this comment obsolete as well with @laanwj's remark below?

Reply:
I was suggesting to put the invalid method between valid methods.

Reply:
I see, that makes sense - changed.